roles/openshift_aws/tasks/scale_group.yml

```diff
@@ -25,6 +25,7 @@
           | combine(openshift_aws_node_group.tags)
           | combine({'deployment_serial': l_deployment_serial, 'ami': openshift_aws_ami_map[openshift_aws_node_group.group] | default(openshift_aws_ami)})
           | combine({'openshift-node-group-config': openshift_aws_node_group.node_group_config | default('unset') }) }}"
+    - Name: "{{ openshift_aws_clusterid }}"
```

Comment:
I think this should also include the node's role (i.e., compute, infra, master).